plone/dexterity/schema.py

```diff
@@ -49,10 +49,13 @@ def volatile(func):
     def decorator(self, portal_type):
         """lookup fti from portal_type and cache
         """
-        if IDexterityFTI.providedBy(portal_type):
-            fti = portal_type
+        if portal_type is not None:
```

Comment:
well, in fact, the check would be if it's not a string, see https://github.com/zopefoundation/zope.interface/commit/81f50ca16007d0eb6beb7a51b5bf32b6f1cb410a#diff-f68fd39e6cf94748a382d0d560feb115R887
but I fear it is more expensive, and in our case, None is the only value that may pass in.